fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/election/OpenElection.java

```diff
@@ -0,0 +1,78 @@
+package com.github.dedis.popstellar.model.network.method.message.data.election;
+
+import static com.github.dedis.popstellar.model.objects.event.EventState.CREATED;
+
+import androidx.annotation.NonNull;
+
+import com.github.dedis.popstellar.R;
+import com.github.dedis.popstellar.model.network.method.message.data.Action;
+import com.github.dedis.popstellar.model.network.method.message.data.Data;
+import com.github.dedis.popstellar.model.network.method.message.data.Objects;
+import com.google.gson.annotations.SerializedName;
+
+public final class OpenElection extends Data {
+
+  @SerializedName("lao")
+  private final String laoId;
+
+  @SerializedName("election")
+  private final String electionId;
+
+  @SerializedName("opened_at")
+  private final long openedAt;
+
+  public OpenElection(@NonNull String laoId, @NonNull String electionId, long openedAt) {
+    this.laoId = laoId;
+    this.electionId = electionId;
+    this.openedAt = openedAt;
+  }
+
+  @Override
+  public String getObject() {
+    return Objects.ELECTION.getObject();
+  }
+
+  @Override
+  public String getAction() {
+    return Action.OPEN.getAction();
+  }
+
+  public String getLaoId() {
+    return laoId;
+  }
+
+  public String getElectionId() {
+    return electionId;
+  }
```

Comment:
And here

Reply:
You can annotate `getLaoId()` and `getElectionId()` if you really want to use the NonNull annotation to the fullest.
But this is nitpicking.